xml/admin_rbd.xml

```diff
@@ -1130,7 +1132,7 @@ EOF</screen>
      <command>feature enable</command> subcommand, the pool and image name, and
      the feature name:
     </para>
-<screen>&prompt.cephuser;rbd --cluster local feature enable <replaceable>POOL_NAME</replaceable>/<replaceable>IMAGE_NAME</replaceable> journaling</screen>
+<screen>&prompt.cephuser;rbd --cluster local feature enable <replaceable>POOL_NAME</replaceable>/<replaceable>IMAGE_NAME</replaceable> journal</screen>
```

Comment:
neither rivals nor man page talks about 'journal' parameter, the original was correct
```suggestion
<screen>&prompt.cephuser;rbd --cluster local feature enable <replaceable>POOL_NAME</replaceable>/<replaceable>IMAGE_NAME</replaceable> exclusive-lock
&prompt.cephuser;rbd --cluster local feature enable <replaceable>POOL_NAME</replaceable>/<replaceable>IMAGE_NAME</replaceable> journaling</screen>
```

Reply:
Apply when you're ready :)